src/server/socketHandlers/scGraphQLHandler.js

```diff
@@ -3,10 +3,9 @@ import Schema from 'server/graphql/rootSchema';
 
 // eslint-disable-next-line no-underscore-dangle
 const mutations = Schema._mutationType && Schema._mutationType._fields || {};
-const whitelist = ['updateUserProfile', 'summarizeMeeting'];
+const whitelist = ['createUserPicturePutUrl', 'updateUserProfile', 'summarizeMeeting'];
```

Comment:
Very reasonable design choice. From the standpoint of the developer user-experience, it took me about 30 minutes to debug why my request was resolving, but not making it's way back to the client (had to check mutationHandler, then work backward to the backend). If others start extending our app, I suspect they may run into something similar.

What if we made this more explicit? Like a meta flag on the mutation options or something similar?

```javascript
return cashay.mutate('mutation', {meta: {noResponseData: true}});
```

Meta options might be useful to implement server-specific behavior...